### ISSUE TITLE
Property 'onClick' is missing

### ISSUE THREAD

**Issue by muuvmuuv** [RoccoC/webpack-build-notifier#38]
Why do I need to handle onClick by myself? In the docs it says ["by default"](https://github.com/RoccoC/webpack-build-notifier#onclick) so I guess this should be optional:

```bash
Argument of type '{ title: string; }' is not assignable to parameter of type 'WebpackBuildNotifierConfig'.
  Property 'onClick' is missing in type '{ title: string; }' but required in type 'WebpackBuildNotifierConfig'.
```

see: [index.d.ts#L96](https://github.com/RoccoC/webpack-build-notifier/blob/master/index.d.ts#L96)

**Comment by RoccoC** [comment]
You are right, `onClick` should be made optional.

**Comment by RoccoC** [comment]
Fixed in 0.1.32.
